tests/test_single_node.py

```diff
@@ -19,22 +18,44 @@ def test_block_production(wallet: Wallet, node: Node):
     assert block['witness'] == node.config['witness'][1:-1]
 
 
-def test_genesis_block(wallet: Wallet, genesis: Genesis):
+def test_genesis_block(wallet: Wallet, genesis: Genesis, creator: str):
     info = wallet.get_dynamic_global_properties()
 
-    expected_total_supply = (Amount(genesis['accounts_supply']) +
-                             Amount(genesis['rewards_supply']) +
-                             Amount(genesis['registration_supply']) +
-                             Amount(genesis['founders_supply']) +
-                             Amount(genesis['steemit_bounty_accounts_supply']) +
-                             Amount(genesis['development_sp_supply']) +
-                             Amount(genesis['development_scr_supply']))
+    records = [
+        'accounts_supply', 'rewards_supply', 'registration_supply',
+        'founders_supply', 'steemit_bounty_accounts_supply',
+        'development_sp_supply', 'development_scr_supply'
+    ]
+
+    expected_total_supply = Amount()
+
+    for r in records:
+        expected_total_supply = expected_total_supply + Amount(genesis[r])
 
     assert Amount(info['total_supply']) == expected_total_supply
-    assert wallet.get_account_scr_balance(initdelegate.name) == Amount('80.000000000 SCR')
+    assert wallet.get_account_scr_balance(creator) == Amount('80.000000000 SCR')
```

Comment:
add balance check for all account in genesis

Reply:
added